samples/apps/copilot-chat-app/webapi/CopilotChat/Controllers/ChatController.cs

```diff
@@ -61,7 +66,8 @@ public ChatController(ILogger<ChatController> logger)
     [ProducesResponseType(StatusCodes.Status404NotFound)]
     public async Task<IActionResult> ChatAsync(
         [FromServices] IKernel kernel,
-        [FromServices] CopilotChatPlanner planner,
+        [FromServices] IHubContext<MessageRelayHub> messageRelayHubContext,
+        [FromServices] CopilotChatPlanner? planner,
```

Comment:
I didn't think we could have an optional parameter coming from services. Do we need it to be nullable?

Reply:
I'd remove it as optional if you're going to force it as a non-null in line 84